content/post/go-safehtml.md

```diff
@@ -63,10 +57,22 @@ This is the most nuanced matter. Sometimes the safehtml API is too inconvenient
 
 For these **very rare** situations you can use this package. Importing it should be restricted to a hand-picked set of dependants and every new import should require some security-aware folks to review it.
 
-Usages of this package are your single point of failure, so make sure you only use it when necessary. (This sentence assumes you will eventually get rid of legacy conversions)
+Make sure that the usage **is safe and will stay safe**, as uncheckedconversions do not increase safety. They are just there to inform the compiler that you have reviewed the code and want it to be trusted. Follow these guidelines:
```

Comment:
Added this section based mostly on go/hardening-comments guidance I've written recently.